src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/Symbols.vb

```diff
@@ -932,6 +940,9 @@ Namespace Microsoft.VisualBasic.CompilerServices
                 Return result
             End Function
 
+            <UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:UnrecognizedReflectionPattern",
+                Justification:="_type is annotated with .All, so it's BaseType will be annotated as well and it is safe to call GetMember on the BaseType.
```

Comment:
I'm not sure I understand this message. Where is `BaseType` used?

Reply:
Apologies as it is not straight forward to figure this out. The variable classConstraint is created by calling `GetClassConstraint(_type)`, which if you see it's implementation, all it's doing is:

https://github.com/dotnet/runtime/blob/ac82799250fe42c8ba2941aef487aca94ecf04cc/src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/Symbols.vb#L522-L531

which basically means it is calling BaseType on `_type` and then the linker is warning because we are then calling GetMember into that and we haven't annotated the basetype, but because of the fact that `_type` itself is annotated with `DAM.All` then it's basetype and members should be preserved too.